src/environments/environment.test.ts

```diff
@@ -1,8 +1,9 @@
-import { name, version } from '../../package.json';
+import * as packageInfo from '../../package.json';
+const { name, version } = packageInfo;
```

Comment:
Not a blocker, but is there a reason for importing everything from `package.json` instead of deestructuring just what we need?

Reply:
I was getting an error when trying to access package variables that way. I don't recall the error though I'll have to find it for you.

Reply:
I believe this is no longer the way to access `package.json` variables. It has been deprecated

Reply:
Interesting. I'll have to look into it as well because we are still doing it the old way in the React boilerplate.